### ISSUE TITLE
[package] openssl/1.1.1l: OUTDATED CODE

### ISSUE THREAD

**Issue by norasyeezys** [conan-io/conan-center-index#22734]
### Description

Openssl features outdated code and I can't find a workaround because openssl has so many versions.

### Package and Environment Details

* Package Name/Version: **openssl/1.1.1l**
* Operating System+version: **Mac OS Monterey 12.5**
* Compiler+version: **GCC 8**
* Docker image: **conanio/gcc8**
* Conan version: **conan 2.0.17**
* Python version: **Python 3.12.1**

### Conan profile

[settings]
os=Macos
os_build=Macos
arch=armv8
arch_build=armv8
compiler=apple-clang
compiler.cppstd=gnu17
compiler.version=14
compiler.libcxx=libc++
build_type=Release
[options]
[conf]
[build_requires]
[env]

### Steps to reproduce

I just ran: ./run.sh on a file with Conan on it

### Logs

<details><summary>Click to expand log</summary>

```
ERROR: Package 'openssl/1.1.1l' not resolved: openssl/1.1.1l: Cannot load recipe.
Error loading conanfile at '/.../.conan2/p/opens7d6da805c1982/e/conanfile.py': Unable to load conanfile in /.../.conan2/p/opens7d6da805c1982/e/conanfile.py
  File "<frozen importlib._bootstrap_external>", line 994, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/.../.conan2/p/opens7d6da805c1982/e/conanfile.py", line 4, in <module>
    from conans import ConanFile, AutoToolsBuildEnvironment, tools
ImportError: cannot import name 'ConanFile' from 'conans' (/opt/homebrew/Cellar/conan/2.0.17_1/libexec/lib/python3.12/site-packages/conans/__init__.py)
```

</details>

**Comment by Ahajha** [comment]
If you still need a version from 1.1.1, it looks like 1.1.1t/u/v/w are the only versions currently supported by Conan 2.0: https://conan.io/center/recipes/openssl

**Comment by Croydon** [comment]
Yes, only 1.1.1w is still maintained as a Conan package. Please switch to it.

The error you are getting tells you that the other version does not support Conan 2.

Issue can probably be closed as there is nothing to do here from our side.
